coppolaemilio/dialogic
# Audacity label importer

This is a tool to mainly improve the voice acting support of the text events, but applies to all audio events that feature a start_time and stop_time value.
A developer may now use the label feature in audacity to mark where a line will begin and end. This data may be exported from Audacity to a text file.
This text file may now be imported by this tool.

how to use/test:

1. How to use open any event that has an audio picker.
2. Unfold the audiopicker's advanced settings, find the start and end section.
3. press the new Aud butto
3a. If no label track have been loaded, the file loader will appear.
3b. select a label track export txt file 
3c. press the Aud button again
4. select a timecode.
5. observe that the audiopicker's start and end values change to the selected item
6. press the LOAD NEW LABEL TRACK button to load a different label track (returning to step 3a)
7. Do not let yourself be caught in an infinite loop by step 6

known issues:
- The popup's tool panel is transparent. Tried to fix it with a panel, but it somehow failed.
- The popup appears at the wrong place. Possible that it needs an offset I do not know about.
- Attempting to drag the scrollbar closes the window.
